src/apb/engine.py

```diff
@@ -1047,8 +1045,6 @@ def cmdrun_push(**kwargs):
     blob = base64.b64encode(spec)
     data_spec = {'apbSpec': blob}
     broker = kwargs["broker"]
-    if broker is None:
-        broker = get_asb_route()
```

Comment:
@dymurray is this removal correct here? maybe this is what's making `--broker` required

Reply:
Ah no, it seems it's being called from `broker_request()`

Reply:
@paolobueno Yeah there seems to be something that the openshift restclient is complaining about when running `get_asb_route()` from `broker_request()`.